index.d.ts

```diff
@@ -39,6 +39,7 @@ declare namespace NodePolyfillPlugin {
 		By default, the modules that were polyfilled in Webpack 4 are mirrored over. However, if you don't want a module like console to be polyfilled you can specify alises to be skipped here.
 		*/
 		excludeAliases?: readonly Alias[]
+                includeAliases?: readonly Alias[]
```

Comment:
Use `MergeExclusive` from [`type-fest`](https://github.com/sindresorhus/type-fest) for more accurate typings.